src/components/usa-icon/usa-icon--sizes/usa-icon--sizes.stories.js

```diff
@@ -0,0 +1,12 @@
+import "../../../stylesheets/uswds.scss";
+import Component from "./usa-icon--sizes.twig";
+import Data from "./usa-icon--sizes.json";
+
+export default {
+  title: "Components/Icon Sizes",
+};
```

Comment:
Potentially something that is saved for a later phase, but wondering if we want to hide controls that the user won't want to actually control (like the icons img path control on this story) with table: { disable: true }